### PR TITLE
Reduce Console Errors

### PR DESCRIPTION
**Pull request by Luke-W-Hart**
Reference #2289 
Addresses this error...
Screenshot from 2024-06-17 15-41-44

Navigation by the header tabs and side nav links still works, but I'm not sure if removing the innerRefs breaks anything else.

### CODE REVIEW

File: src/components/Header/index.jsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ const headerLogo64dpi = 'images/gc_' + angleMode + '_yellow_logo_64.png';
 const headerLogo56dpi = 'images/gc_' + angleMode + '_yellow_logo_56.png';
 const headerLogo56dpiNoText = 'images/gc_' + angleMode + '_yellow_logo_56_vert.png';
 
-const ForwardNavLink = forwardRef((props, ref) => <NavLink innerRef={ref} {...props} />);
+const ForwardNavLink = forwardRef((props, ref) => <NavLink {...props} />);
```
Comment (EjPlatzer) [inline src/components/Header/index.jsx:30]: I think this became unnecessary when we upgraded react-router past version 5.1, but no one has removed the unnecessary ref forwarding.

At this point, `ForwardNavLink` is at best useless and at worst silently swallowing the `ref`, so we should get rid of it and just pass a plain old `NavLink` where it was used.

File: src/components/Nav/components/NavAvatar/index.jsx [hunk 2]
```diff
@@ -42,7 +42,6 @@ const GordonNavAvatar = ({ onLinkClick }) => {
   const buttonLink = forwardRef((props, ref) => (
     <Link
       {...props}
-      innerRef={ref}
       to={profile ? `/myprofile` : '/'}
```
Comment (EjPlatzer) [inline src/components/Nav/components/NavAvatar/index.jsx:45]: As above, we should just get rid of `forwardRef` here and use a plain old `Link`.

**Review by EjPlatzer — CHANGES_REQUESTED** [review]
Nice work! Thanks for doing this useful maintenance. I think this has knock on opportunities to simplify the code even further.

**Comment by Luke-W-Hart** [comment]
Should be good now, and also added a key to the event header columns to resolve this error...
Screenshot from 2024-06-17 16-27-05

Thanks for the help Evan!

**Review by EjPlatzer — APPROVED** [review]
LGTM!

**Pull request merged**